src/utils/searchAndSort.ts

```diff
@@ -78,11 +80,14 @@ const searchConditionMet = <T extends Record<string, unknown>>(result: T, condit
       return searchValues.some((value) => resultValue.includes(value));
     } else if (condition.type === 'Fuzzy') {
       return searchValues.some((searchValue) => {
+        const exactValue = removeDoubleQuotes(searchValue);
```

Comment:
If the desired behavior is "exact exact" instead of contains "exact", we can move this chunk out before Fuzzy, and check for equality.